### PR TITLE
fix(CLI): Fixing and ignoring intermittent tests

### LINKED ISSUE #26632: CLI: Fix blinking test SiteCommandIT
### Parent Issue

_No response_

### Problem Statement

SiteCommandIT fails randomly. 

Screenshot 2023-11-07 at 5 13 01 PM

### Steps to Reproduce

Run this test multiple times in cloud

### Acceptance Criteria

The test should always pass

### dotCMS Version

master

### Proposed Objective

Core Features

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### LINKED ISSUE #26633: CLI: Fix blinking test SiteAPIIT
### Parent Issue

_No response_

### Problem Statement

SiteAPIIT fails randomly.

Screenshot 2023-11-07 at 5 12 21 PM

### Steps to Reproduce

Run this test multiple times in cloud.

### Acceptance Criteria

The test should always pass.

### dotCMS Version

master

### Proposed Objective

Core Features

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by jgambarios**
This PR resolves some intermittent tests and ignores many others until a cause and solution are found.

### CODE REVIEW

File: tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/common/SiteTestHelper.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                                        response.entity().isWorking()))) {
+                    return true;
+                }
+            } catch (NotFoundException e) {
+                // Do nothing
+            }
+
+            try {
+                Thread.sleep(2000); // Sleep for 2 second
+            } catch (InterruptedException ex) {
```
Comment (spbolton) [inline tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/common/SiteTestHelper.java:54]: We need to get away from the anti pattern of using Thread.sleep in tests.   Should use the Awaitility library instead.  We have it in master. 

https://www.baeldung.com/java-thread-sleep-vs-awaitility-await

If changing is going to delay getting master working then look at this when you come back around.

**Review by erickgonzalez — APPROVED** [review]

File: tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/common/SiteTestHelper.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+     * Checks if a site with the given name exists.
+     *
+     * @param siteName the name of the site to check
+     * @param siteAPI  the site api to use
+     * @return true if the site exists, false otherwise
+     */
+    protected Boolean siteExist(final String siteName, final SiteAPI siteAPI) {
+
+        long start = System.currentTimeMillis();
+        long end = start + 15 * 1000; // 15 seconds * 1000 ms/sec
```
[the remark this answers is not in the document]
Comment (fabrizzio-dotCMS) [reply tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/common/SiteTestHelper.java:35]: You can also achieve more readable code through the class TimeUnit by doing this
TimeUnit.SECONDS.toMillis(15)

**Review by fabrizzio-dotCMS — APPROVED** [review]

File: tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/common/SiteTestHelper.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (jgambarios) [reply tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/common/SiteTestHelper.java]: Done

File: tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/common/SiteTestHelper.java (hunk 1 above)
Comment (jgambarios) [reply tools/dotcms-cli/api-data-model/src/test/java/com/dotcms/common/SiteTestHelper.java]: Removed `Thread.sleep`, now we are using `org.testcontainers.shaded.org.awaitility.Awaitility.await`.

**Pull request merged**